src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,90 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
+    private static final int MIN_ID_LENGTH = 1;
+    private static final int MAX_ID_LENGTH = 200;
```

Comment:
The constants `MIN_ID_LENGTH` and `MAX_ID_LENGTH` are used for user ID validation, but they seem to represent length constraints. However, user IDs are typically numeric and should be validated for range rather than length. Consider renaming these constants to `MIN_ID_VALUE` and `MAX_ID_VALUE` for clarity.